packages/mdc-select/README.md

```diff
@@ -242,113 +183,36 @@ is outlined below.
-| `deregisterMenuInteractionHandler(type: string, handler: EventListener) => void` | Opposite of `registerMenuInteractionHandler`. |
-| `notifyChange() => void` | Broadcast a change event, similar to the `change` event emitted by an `HTMLSelectElement`. While we use custom events in our implementation for this, you can use any mechanism desired for notifications, such as callbacks, reactive streams, etc. Note that you can also pass data within your event if you'd like via `foundation.getValue()` and `foundation.getSelectedIndex()`. |
-| `getWindowInnerHeight() => number` | Returns the `innerHeight` property of the `window` element. |
-
-### The full foundation API
-
-#### MDCSelectFoundation.getValue() => string
-
-Returns the value of the currently selected option, or an empty string if no option is selected.
-
-#### MDCSelectFoundation.getSelectedIndex() => number
-
-Returns the index of the currently selected option. Returns -1 if no option is currently selected.
-
-#### MDCSelectFoundation.setSelectedIndex(selectedIndex: number) => void
-
-Sets the selected index of the component.
-
-#### MDCSelectFoundation.isDisabled() => boolean
-
-Returns whether or not the select is disabled.
-
-#### MDCSelectFoundation.setDisabled(disabled: boolean) => void
-
-Enables/disables the select.
-
-## Theming
-
-The select's bottom border is set to the current theme's primary color when focused.
+| `floatLabel(value: boolean) => void` | Floats or defloats label as necessary. |
+| `activateBottomLine() => void` | Activates the bottom line component. |
+| `deactivateBottomLine() => void` | Deactivates the bottom line component. |
+| `setDisabled(disabled: boolean) => void` | Sets the `disabled` property of the `<select>` element. |
+| `registerInteractionHandler(type: string, handler: EventListener) => void` | Adds an event listener `handler` for event type `type` on the `<select>` element. |
+| `deregisterInteractionHandler(type: string, handler: EventListener) => void` | Removes an event listener `handler` for event type `type` on the `<select>` element. |
+| `getSelectedIndex() => number` | Returns the selected index of the `<select>` element. |
+| `setSelectedIndex(index: number) => void` | Sets the select's selectedValue to the option found at the provided index. If the index is out of the select's range, it will default to -1. |
+| `getValue() => string` | Returns the value selected on the `<select>` element. |
+| `setValue(value: string) => void` | Sets the select's value. If no option has the provided value, it sets `value` to empty string. |
```

Comment:
Reduce this to: Sets the value of the `<select>` element.

(The rest is re-documenting native behavior.)